Explorer/Assets/DCL/WebRequests/WebContentSizes/Playground/WebContentsSizesPlayground.unity

```diff
@@ -0,0 +1,427 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
```

Comment:
I am wondering if all Playground related folders are in the Editor only assembly

Reply:
I didn't get the question